pages/_app.tsx

```diff
@@ -2,4 +2,13 @@ import App from "next/app";
 
 import "normalize.css";
 
+import Store from "../store/Store";
+
+// Only initialise store if on client. IndexedDB is not available when only
+// server side, so we need to make sure we're client side before trying to
+// initialise our DB.
```

Comment:
The main point of this is that the user's data doesn't exist on the server, so there's no need to attempt to access it. If we needed it, we'd polyfill IndexedDB, rather than saying "doesn't exist, so won't bother".